src/System.Net.Requests/ref/System.Net.Requests.cs

```diff
@@ -124,3 +124,59 @@ public abstract partial class WebResponse : System.IDisposable
         public abstract System.IO.Stream GetResponseStream();
     }
 }
+namespace System.Net.Cache
```

Comment:
No it does not.  Those failures you are pointing out would be hit if that XML project was targeting netstandard 1.7 just the same.  The project is defining types that are now being exposed.